api/workers/autoscale.go

```diff
@@ -93,8 +95,12 @@ func autoscaleRack() {
 		system.Count = desired
 	}
 
-	log.Logf("change=%d", (desired - system.Count))
+	// nothing changed, return
+	if system.Count == oldCount {
+		return
+	}
 
+	log.Logf("change=%d", (system.Count - oldCount))
```

Comment:
Probably just switch the log to just above here. So we still see the change=0 log.